luda-editor/luda-editor-client/src/app/editor/job/move_camera_clip.rs

```diff
@@ -11,139 +11,142 @@ pub struct MoveCameraClipJob {
 
 impl JobExecute for MoveCameraClipJob {
     fn execute(&self, sequence: &Sequence) -> Result<Sequence, String> {
-        let mut sequence = sequence.clone();
-        let track = find_camera_track_of_clip(&self.clip_id, &sequence);
-        if track.is_none() {
-            return Err(format!("camera track not found"));
+        let sequence = sequence.clone();
+        let camera_track_id = get_camera_clip_id(&sequence);
```

Comment:
```suggestion
        let camera_track_id = get_camera_track_id(&sequence);
```
related to https://github.com/NamseEnt/namseent/pull/96/files#r796675567